pylib/Stages/MTTDefaults/DefaultMTTDefaults.py

```diff
@@ -36,7 +36,7 @@ def __init__(self):
         MTTDefaultsMTTStage.__init__(self)
         self.options = {}
         self.options['trial'] = (False, "Use when testing your MTT client setup; results that are generated and submitted to the database are marked as \"trials\" and are not included in normal reporting.")
-        self.options['scratchdir'] = ("./mttscratch", "Specify the DIRECTORY under which scratch files are to be stored")
+        self.options['scratchdir'] = (None, "Specify the DIRECTORY under which scratch files are to be stored")
```

Comment:
I'm trying to wrap my head around this change. If we have no ```scratchdir``` defined, then we somewhere have to define a default fallback value as we need a scratch directory to perform MTT. You can always override the option with the cmd line - we have a parsing operation for just that purpose - and you are welcome to add envar detection/override if you like.

So why are we now setting the default value to ```None``` when that value is not valid?